third_party/CMakeLists.txt

```diff
@@ -88,7 +88,7 @@ if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/highway/CMakeLists.txt" AND
   configure_file("${CMAKE_CURRENT_SOURCE_DIR}/highway/LICENSE"
                  ${PROJECT_BINARY_DIR}/LICENSE.highway COPYONLY)
 else()
-  pkg_check_modules(HWY libhwy)
+  pkg_check_modules(HWY libhwy>=0.15.0)
```

Comment:
ahh, this doesn't work on ubuntu 18.04 because the cmake there is too old. let's remove this line for now.

Reply:
OK, turned it into a comment.